README.md

```diff
@@ -166,11 +166,9 @@ use of the software operates as your consent to these practices.
 
 ### Telemetry Configuration
 
-Telemetry collection is off by default.
+Telemetry collection is on by default.
 
-To opt in, please run `pf config set cli.telemetry_enabled=true` to turn it on.
-
-For Europe user, please run `pf config set cli.eu_user=true` to make sure telemetry is sent to Europe server.
+To opt out, please run `pf config set cli.telemetry_enabled=false` to turn it off.
```

Comment:
Aslo add sth like this: 
![image](https://github.com/microsoft/promptflow/assets/47586720/34aa6a7d-adcc-4b7d-b099-cf8d5e769e51)

Reply:
Added in `pf`
1st time run:
![image](https://github.com/microsoft/promptflow/assets/7776147/f6232e53-dd92-4203-bfe5-3e22aaacc293)

2nd time run:
![image](https://github.com/microsoft/promptflow/assets/7776147/796c28a0-1660-43f0-be15-1a7317cca224)

Reply:
put the ascii art to top

Reply:
Change to this
![image](https://github.com/microsoft/promptflow/assets/7776147/e4cac5d5-a10d-45a4-85aa-62baaf3f6b0d)